packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
 
-const autocompleteDataReceivedEvent = new CustomEvent('autocomplete_received_results',
+// @TODO allow for sending event with searchbar ID. This should allow for one
+// story's events to not affect another story.
+const autocompleteDataReceivedEvent = new CustomEvent('sfx::autocomplete_received_results',
   { detail: [
       { "title": "Brands", "items": [{ "label": "Cats" }, { "label": "Dogs" }] },
       { "title": "default", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }
     ],
     bubbles: true }
 );
 
+function getSayt(searchbar='', showSayt=true): string {
+  const showAttribute = boolean('visible', showSayt) ? 'visible' : '';
+  const closeText = text('Close link text', 'Close');
+  const showCloseButton = boolean('Show Close button', true) ? 'showclosebutton' : '';
+
+  return `<sfx-sayt${ searchbar ? ` searchbar="${ searchbar }"` : '' }
+  closetext="${ closeText }"${ showCloseButton ? `
+  ${ showCloseButton }` : ''}${ showAttribute ? `
+  ${ showAttribute }` : ''}
```

Comment:
Is there a way we can format this better?

Reply:
I don't think so, but I'm open to suggestions. The way it's done here is very intentional.

It's possible to use HTML comments to allow for code line breaks without visual line breaks, but I don't really want to inject comments all over the code (especially since it will be output on each story).

Reply:
Using escape sequences (e.g. `\n`) might help.

Reply:
The problem is too much whitespace, rather than not enough. I can use `\n` to produce empty lines between each piece, but I'll end up putting most of it on one line.

As soon as one attribute ends, I need to begin a new one to prevent extra newlines being added.

If you have an idea, feel free to take a shot at it, but I won't spend additional time on it unless I have a concrete plan.

Reply:
Option 1: "hide the whitespace in the interpolated area"

```js
  return \
`<sfx-sayt${
  searchbox ? `\n  searchbox="${searchbox}"` : ''}
  closetext="${closeText}"${
  showCloseButton ? `\n  ${showCloseButton}` : ''}${
  showAttribute ? `\n  ${showAttribute}` : ''}
></sfx-sayt>`;
```

Option 2: "string concatenation is your friend"

```js
return \
'<sfx-sayt\n'
+ (searchbox ? `  searchbox="${searchbox}"\n` : '')
+ `  closetext="${closeText}"\n`
+ (showCloseButton ? `  ${showClosebutton}\n` : '')
+ (showAttribute ? `  ${showAttribute}\n` : '')
+ '></sfx-sayt>'
```

I'm somewhat partial to option 2, although it doesn't show the indentation as well as option 1.

Reply:
The backslash causes errors, but option 2 otherwise works.